Makefile

```diff
@@ -68,18 +68,22 @@ check-install: ## Checks that you've installed this repository correctly
 .PHONY: manager
 manager: generate  ## Build manager binary.
 	bazel build //cmd/manager $(BAZEL_ARGS)
+	install bazel-bin/cmd/manager/*/manager $(shell go env GOPATH)/bin/aws-manager
```

Comment:
Just noticed the wildcard here, this will cause issues if someone has run the release-binaries target and cross compiled binaries for other targets.

Reply:
ah hmm so it will

Reply:
doh, didn't beat the merge...